packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js

```diff
@@ -0,0 +1,79 @@
+import counterpart from 'counterpart'
+import React from 'react'
+import PropTypes from 'prop-types'
+import { inject, observer, PropTypes as MobXPropTypes } from 'mobx-react'
+import styled from 'styled-components'
+import { Box, Paragraph } from 'grommet'
+import { Markdownz, Media } from '@zooniverse/react-components'
+import StepNavigation from './components/StepNavigation'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function storeMapper(stores) {
+  const { stepWithMedium } = stores.classifierStore.tutorials
+  return {
+    stepWithMedium
+  }
+}
+
+const StyledMarkdownWrapper = styled(Box)`
+  height: 100%;
+  max-height: ${props => props.isThereMedia ? '150px' : '100%'} ;
+`
+
+@inject(storeMapper)
+@observer
+class SlideTutorial extends React.Component {
+  render() {
+    const { stepWithMedium } = this.props
+    if (stepWithMedium && Object.keys(stepWithMedium).length > 0) {
+      const { medium, step } = stepWithMedium
+      const isThereMedia = medium && medium.src
+      return (
+        <Box
+          height="100%"
+          justify='between'
+          pad='medium'
+        >
+          {isThereMedia &&
+            <Media alt='' fit='contain' height={200} src={medium.src} />}
+          <StyledMarkdownWrapper isThereMedia={isThereMedia} overflow='auto'>
+            <Markdownz>{step.content}</Markdownz>
```

Comment:
Can we start flagging up where English language strings are used in the code? This will save a lot of work later, when these will need to be found and replaced with strings from the resource translation.

Reply:
Yes absolutely.

Reply:
Would a running comment in the translation issue including a list with the file name and line number be sufficient?

Reply:
That would work but a comment in the code, that we could search for, might be more convenient. Then we could search across the code for that text and replace the existing code with language strings from the translations.

Reply:
How about using `//TODO: ` comments in the code when we know we'll have to come back to something?